### ISSUE TITLE
_a.errors.clear is not a function

### ISSUE THREAD

**Issue by florian-lefebvre** [andrewcourtice/harlem#59]
**Describe the bug**
When using the actions extension, calling any action throws the following error:
```bash
runtime-core.esm-bundler.js:38 [Vue warn]: Unhandled error during execution of native event handler 
  at <NavBar> 
  at <Index> 
  at <App key=1 > 
  at <NuxtRoot>
runtime-core.esm-bundler.js:218 Uncaught TypeError: _a.errors.clear is not a function
    at index.ts:114:98
    at Store.mutate (store.ts:204:22)
    at mutation (store.ts:229:37)
    at Store.write (store.ts:231:60)
    at clearErrors (index.ts:114:20)
    at Proxy.<anonymous> (index.ts:146:21)
    at _createElementVNode.onClick._cache.<computed>._cache.<computed> (NavBar.vue:118:42)
    at callWithErrorHandling (runtime-core.esm-bundler.js:155:22)
    at callWithAsyncErrorHandling (runtime-core.esm-bundler.js:164:21)
    at HTMLButtonElement.invoker (runtime-dom.esm-bundler.js:369:13)
```

**To Reproduce**
https://github.com/florian-lefebvre/nuxt3-starter-medusa/tree/04f7fafa1afd40259b0af303afe2d8e6ad2ef226

Files to look at are `/stores/useStore.ts` (and the `init` action) and `/components/Layout/NavBar.vue`.

Steps to reproduce the behaviour:
1. Install the dependencies (`yarn`)
2. Start the Nuxt3 app (`yarn dev`)
3. Click on **INIT** at the top

**Expected behaviour**
Nothing should happen, no errors.

**Version (please complete the following information):**
 - **Harlem:** 2.3.3
 - **Nuxt:** 3.0.0-rc.1
 - **Runtime Environment:** Brave V1.38.109 on Windows 10

**Comment by andrewcourtice** [comment]
Hi @florian-lefebvre 

Thanks for reporting this.

It looks to be occurring due to the way the application is being compiled. My suspicion is that the build target is pre-ES6. A key giveaway is that the action state that is getting created by Harlem uses native ES6 maps for storing errors and action instances. Somewhere along the line the errors and instances are getting created as objects as opposed to native maps.

Here is the code that creates the action state:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/11718453/166344616-f6c68223-adad-41a6-8868-272feb94b4c7.png">
note the use of `new Map()`

And here is what is getting produced:
<img width="217" alt="image" src="https://user-images.githubusercontent.com/11718453/166344559-b1482d9a-4365-494f-9b81-9d653b57b231.png">
Note how the `error` and `instances` properties are now objects, not maps.

I'll keep having a look into it but at first glance it appears to be a compilation problem.

**Comment by andrewcourtice** [comment]
@florian-lefebvre Actually, I've found the issue. Working on a fix...

**Comment by florian-lefebvre** [comment]
Awesome, thanks!

**Comment by andrewcourtice** [comment]
@florian-lefebvre This should be fixed in version [2.3.5](https://github.com/andrewcourtice/harlem/releases/tag/v2.3.5)
